readthedocs.yaml

```diff
@@ -5,6 +5,15 @@ build:
   tools:
     python: "mambaforge-4.10"
 
+  jobs:
+    pre_build:
+      - pip install awscli
```

Comment:
let's move this `pip install awscli` here: https://github.com/ploomber/sklearn-evaluation/blob/21c6fb5ab626e2b1f2ef00b671763ee6812f2b96/doc/environment.yml#L27 I think it'll speed things up